src/open_inwoner/pdc/managers.py

```diff
@@ -114,7 +114,7 @@ def filter_by_zaken(self, cases: List[Zaak]):
         pks = []
         for category in qs:
             for identificatie in category.zaaktypen:
-                if not identificatie in months_since_last_zaak_per_zaaktype:
```

Comment:
Bug?

Reply:
This is due to https://www.flake8rules.com/rules/E713.html. `if not var in list` could be weird to read if var is a boolean

Reply:
I thought the original was also not functional the same but apparently it is.